From 37001c6eb644c337c64e1e703a43d5b6bfe64676 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 18 Apr 2021 18:31:31 +0200
Subject: [PATCH 2/2] foris-plugins-distutils: remove this package from our
 repository

Since upstream change all host packages are downloaded from PyPI.org and
they need to be specific in packages feed (preferably) in
lang/python/host-pip-requirements/foo.txt

This means that this package which was created to have host package is
no longer necessary and it could save us some minutes while compiling
it. No packages, which you can install on your router depends on it.

See for more details:
https://github.com/openwrt/packages/commit/722a5b8efa19fd54728af462441dce08cd7e545b
---
 web/foris/foris-plugins-distutils/Makefile | 54 ----------------------
 1 file changed, 54 deletions(-)
 delete mode 100644 web/foris/foris-plugins-distutils/Makefile

diff --git a/web/foris/foris-plugins-distutils/Makefile b/web/foris/foris-plugins-distutils/Makefile
deleted file mode 100644
index 9b065fd9a..000000000
--- a/web/foris/foris-plugins-distutils/Makefile
+++ /dev/null
@@ -1,54 +0,0 @@
-#
-# Copyright (C) 2018-2019 CZ.NIC z. s. p. o. (https://www.nic.cz/)
-#
-# This is free software, licensed under the GNU General Public License v3.
-# See /LICENSE for more information.
-#
-
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=foris-plugins-distutils
-PKG_VERSION:=0.3
-PKG_RELEASE:=3
-
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris/foris-plugins-distutils
-PKG_MIRROR_HASH:=529556a48a07df10b02aced0c8541af85c1c4c0ac0975d9b5eabff7791ef396f
-PKG_SOURCE_VERSION:=v$(PKG_VERSION)
-
-PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-PKG_LICENSE:=LICENSE
-PKG_LICENSE_FILES:=GPL-3.0-or-later
-
-HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=libsass
-HOST_BUILD_DEPENDS:=python3/host
-
-include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/package.mk
-include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
-
-define Package/foris-plugins-distutils
-  SECTION:=web
-  CATEGORY:=Web
-  SUBMENU:=Foris
-  TITLE:=Foris Distutils plugins
-  URL:=https://gitlab.nic.cz/turris/foris/foris-plugins-distutils
-  DEPENDS:=+python3-light
-  VARIANT:=python3
-endef
-
-define Package/foris-plugins-distutils/description
-  Distutils to build foris plugins.
-endef
-
-define Host/Compile
-	$(call Build/Compile/HostPy3Mod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
-endef
-
-Host/Install:=
-
-$(eval $(call HostBuild))
-
-$(eval $(call Py3Package,foris-plugins-distutils))
-$(eval $(call BuildPackage,foris-plugins-distutils))
-$(eval $(call BuildPackage,foris-plugins-distutils-src))
-- 
2.25.1

